.github/workflows/hourly.yml

```diff
@@ -0,0 +1,100 @@
+name: Hourly tests 
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+  workflow_dispatch:
+jobs:
+  hourly:
+    name: ${{ matrix.test.name }} (${{ matrix.cluster_type }} ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        rust: [stable]
+        cluster_type: [k8]
+        test:
+          - name: smoke 
+            cmd: cargo run --release --bin flv-test -- smoke --develop --disable-install -- --producer-iteration 100 --producer-record-size 1000 
```

Comment:
can you put producer-iteration and producer recored size as env parameter in the job?